osf/models/node.py

```diff
@@ -2343,13 +2339,6 @@ def is_bookmark_collection(self):
         """For v1 compat"""
         return False
 
-    class Meta:
```

Comment:
Moved this AbstractNode meta, should probs move back.

I can't have `read_node` defined in Meta, for example on both the AbstractNode model and the Node model since they're the same table.  Assume these guardian permissions need to go on the AbstractNode model.